src/SFA.DAS.EmployerIncentives/InnerApi/Requests/Accounts/GetAccountLegalEntityRequest.cs

```diff
@@ -0,0 +1,21 @@
+﻿using SFA.DAS.SharedOuterApi.Interfaces;
+
+namespace SFA.DAS.EmployerIncentives.InnerApi.Requests.Accounts
+{
+    public class GetAccountLegalEntityRequest : IGetApiRequest
```

Comment:
Could this class be renamed? This request gets the "legal entity id", there's already a request defined in other branches (EI-40, EI-145, EI-210) to get the "account legal entity id", could get even more confusing than the fact that there are two ids with similar names :)